*** gram.tab.c	Mon Oct  6 18:38:00 1997
--- mygram.tab.c	Sun Sep 28 13:38:57 1997
***************
*** 177,183 ****
  /* Cause the `yydebug' variable to be defined.  */
  #define YYDEBUG 1
  
! /*  int  yydebug = 1;  */
  
  /* ###################################################### */
  #line 168 "gram.y"
--- 177,183 ----
  /* Cause the `yydebug' variable to be defined.  */
  #define YYDEBUG 1
  
!   int  yydebug = 0;  
  
  /* ###################################################### */
  #line 168 "gram.y"
***************
*** 1053,1086 ****
  #define YYPARSE_PARAM_DECL
  #endif /* not YYPARSE_PARAM */
  
  int
  yyparse(YYPARSE_PARAM_ARG)
       YYPARSE_PARAM_DECL
  {
    register int yystate;
    register int yyn;
-   register short *yyssp;
-   register YYSTYPE *yyvsp;
    int yyerrstatus;	/*  number of tokens to shift before error messages enabled */
    int yychar1 = 0;		/*  lookahead token as an internal (translated) token number */
  
    short	yyssa[YYINITDEPTH];	/*  the state stack			*/
    YYSTYPE yyvsa[YYINITDEPTH];	/*  the semantic value stack		*/
  
-   short *yyss = yyssa;		/*  refer to the stacks thru separate pointers */
-   YYSTYPE *yyvs = yyvsa;	/*  to allow yyoverflow to reallocate them elsewhere */
  
  #ifdef YYLSP_NEEDED
    YYLTYPE yylsa[YYINITDEPTH];	/*  the location stack			*/
-   YYLTYPE *yyls = yylsa;
-   YYLTYPE *yylsp;
  
  #define YYPOPSTACK   (yyvsp--, yyssp--, yylsp--)
  #else
  #define YYPOPSTACK   (yyvsp--, yyssp--)
  #endif
  
-   int yystacksize = YYINITDEPTH;
  
  #ifdef YYPURE
    int yychar;
--- 1053,1088 ----
  #define YYPARSE_PARAM_DECL
  #endif /* not YYPARSE_PARAM */
  
+ 
+ short *yyss;		/*  refer to the stacks thru separate pointers */
+ short *yyssp;
+ YYSTYPE *yyvs, *yyvsp;
+ #ifdef YYLSP_NEEDED
+ YYSTYPE *yyls, *yylsp;
+ #endif
+ int yystacksize;
+ 
  int
  yyparse(YYPARSE_PARAM_ARG)
       YYPARSE_PARAM_DECL
  {
    register int yystate;
    register int yyn;
    int yyerrstatus;	/*  number of tokens to shift before error messages enabled */
    int yychar1 = 0;		/*  lookahead token as an internal (translated) token number */
  
    short	yyssa[YYINITDEPTH];	/*  the state stack			*/
    YYSTYPE yyvsa[YYINITDEPTH];	/*  the semantic value stack		*/
  
  
  #ifdef YYLSP_NEEDED
    YYLTYPE yylsa[YYINITDEPTH];	/*  the location stack			*/
  
  #define YYPOPSTACK   (yyvsp--, yyssp--, yylsp--)
  #else
  #define YYPOPSTACK   (yyvsp--, yyssp--)
  #endif
  
  
  #ifdef YYPURE
    int yychar;
***************
*** 1102,1107 ****
--- 1104,1112 ----
      fprintf(stderr, "Starting parse\n");
  #endif
  
+   yystacksize = YYINITDEPTH;
+   yyvs = yyvsa;	/*  to allow yyoverflow to reallocate them elsewhere */
+   yyss = yyssa;
    yystate = 0;
    yyerrstatus = 0;
    yynerrs = 0;
***************
*** 1116,1121 ****
--- 1121,1127 ----
    yyvsp = yyvs;
  #ifdef YYLSP_NEEDED
    yylsp = yyls;
+   yyls = yylsa;
  #endif
  
  /* Push a new state, which is found in  yystate  .  */
