*** gram.tab.c.orig	Thu Apr 16 15:02:31 1998
--- gram.tab.c	Thu Apr 16 16:38:41 1998
***************
*** 182,188 ****
  /* Cause the `yydebug' variable to be defined.  */
  #define YYDEBUG 1
  
! /*  int  yydebug = 1;  */
  
  /* ###################################################### */
  #line 173 "gram.y"
--- 182,188 ----
  /* Cause the `yydebug' variable to be defined.  */
  #define YYDEBUG 1
  
!   int  yydebug = 0;  
  
  /* ###################################################### */
  #line 173 "gram.y"
***************
*** 1058,1095 ****
  #define YYPARSE_PARAM_DECL
  #endif /* not YYPARSE_PARAM */
  
  int
  yyparse(YYPARSE_PARAM_ARG)
       YYPARSE_PARAM_DECL
  {
-   register int yystate;
    register int yyn;
-   register short *yyssp;
-   register YYSTYPE *yyvsp;
    int yyerrstatus;	/*  number of tokens to shift before error messages enabled */
    int yychar1 = 0;		/*  lookahead token as an internal (translated) token number */
  
    short	yyssa[YYINITDEPTH];	/*  the state stack			*/
    YYSTYPE yyvsa[YYINITDEPTH];	/*  the semantic value stack		*/
  
-   short *yyss = yyssa;		/*  refer to the stacks thru separate pointers */
-   YYSTYPE *yyvs = yyvsa;	/*  to allow yyoverflow to reallocate them elsewhere */
  
  #ifdef YYLSP_NEEDED
    YYLTYPE yylsa[YYINITDEPTH];	/*  the location stack			*/
-   YYLTYPE *yyls = yylsa;
-   YYLTYPE *yylsp;
  
  #define YYPOPSTACK   (yyvsp--, yyssp--, yylsp--)
  #else
  #define YYPOPSTACK   (yyvsp--, yyssp--)
  #endif
  
-   int yystacksize = YYINITDEPTH;
  
  #ifdef YYPURE
    int yychar;
-   YYSTYPE yylval;
    int yynerrs;
  #ifdef YYLSP_NEEDED
    YYLTYPE yylloc;
--- 1058,1100 ----
  #define YYPARSE_PARAM_DECL
  #endif /* not YYPARSE_PARAM */
  
+ 
+ short *yyss;		/*  refer to the stacks thru separate pointers */
+ short *yyssp;
+ YYSTYPE *yyvs, *yyvsp;
+ #ifdef YYLSP_NEEDED
+ YYSTYPE *yyls, *yylsp;
+ #endif
+ int yystacksize;
+ int yystate;
+ 
+ #ifdef YYPURE
+   YYSTYPE yylval;
+ #endif
+ 
  int
  yyparse(YYPARSE_PARAM_ARG)
       YYPARSE_PARAM_DECL
  {
    register int yyn;
    int yyerrstatus;	/*  number of tokens to shift before error messages enabled */
    int yychar1 = 0;		/*  lookahead token as an internal (translated) token number */
  
    short	yyssa[YYINITDEPTH];	/*  the state stack			*/
    YYSTYPE yyvsa[YYINITDEPTH];	/*  the semantic value stack		*/
  
  
  #ifdef YYLSP_NEEDED
    YYLTYPE yylsa[YYINITDEPTH];	/*  the location stack			*/
  
  #define YYPOPSTACK   (yyvsp--, yyssp--, yylsp--)
  #else
  #define YYPOPSTACK   (yyvsp--, yyssp--)
  #endif
  
  
  #ifdef YYPURE
    int yychar;
    int yynerrs;
  #ifdef YYLSP_NEEDED
    YYLTYPE yylloc;
***************
*** 1107,1112 ****
--- 1112,1120 ----
      fprintf(stderr, "Starting parse\n");
  #endif
  
+   yystacksize = YYINITDEPTH;
+   yyvs = yyvsa;	/*  to allow yyoverflow to reallocate them elsewhere */
+   yyss = yyssa;
    yystate = 0;
    yyerrstatus = 0;
    yynerrs = 0;
***************
*** 1121,1126 ****
--- 1129,1135 ----
    yyvsp = yyvs;
  #ifdef YYLSP_NEEDED
    yylsp = yyls;
+   yyls = yylsa;
  #endif
  
  /* Push a new state, which is found in  yystate  .  */
